<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFindDialog</class>
 <widget class="QDialog" name="CFindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="m_pCloseButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="m_pSearchString"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="m_pFindNextButton">
     <property name="text">
      <string>Find Next</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="m_pMatchWholeWord">
     <property name="text">
      <string>Match whole word</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="m_pMatchCase">
     <property name="text">
      <string>Match case</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="m_pSearchDirection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Search direction</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="m_pSearchBackward">
        <property name="text">
         <string>Search Backward</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pSearchForward">
        <property name="text">
         <string>Search Forward</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
